phys/module_ra_effective_radius.F

```diff
@@ -179,9 +179,9 @@ subroutine effectRad(t, qc, nc, qi, qs, qg, rho, qmin, t0c,                 &
 !
      rqc(k) = max(r1,qc(k)*rho(k))
      qcmps(k)  = qc(k)-qccps(k)
-     rqcmps(k) = qcmps(k)*rho(k)
+     rqcmps(k) = max(r1,qcmps(k)*rho(k))
      if(f_qnc) then
-       lammps    = 2.*cdm2*(pidnc*nc(k)/rqcmps(k))**obmr
+       lammps    = 2.*cdm2*(pidnc*MAX(r1,nc(k))/rqcmps(k))**obmr
```

Comment:
did this need a MAX? It seems that 0.**obmr should work.

Reply:
your description implies nc was < 0?

Reply:
@dudhia @weiwangncar 
Jimy, 
Correct. The nc(k) was < 0 (the first value it flagged was order 10^-13). Would it be better to use zero instread of r1 (1.e-12) for the MAX?
New
```
MAX(r1,nc(k))
```
Newer
```
MAX(0.,nc(k))
```

Reply:
@dudhia 
Jimy,
OK, the minimum of the numerator of the positive definite values is not set to zero. These were originally unprotected and could become negative. My first set of mods set them to be no smaller than a scheme-defined small value (1.e-12). Per your suggestions, I have now replaced the nc(k) value with MAX(0.,nc(k)) in both places where this field is used with an exponent.